src/solvers/dg_p4est/containers_2d.jl

```diff
@@ -3,7 +3,7 @@ function init_elements!(elements, mesh::P4estMesh{2}, basis::LobattoLegendreBasi
   @unpack node_coordinates, jacobian_matrix,
           contravariant_vectors, inverse_jacobian = elements
 
-  calc_node_coordinates!(node_coordinates, mesh, basis)
+  calc_node_coordinates!(node_coordinates, mesh, basis.nodes)
```

Comment:
Why this API change?

Reply:
To allow Trixi2Vtk to use this function with visualization nodes.

Reply:
:+1: